force-app/main/triggers/classes/HOT_WorkOrderHandler.cls

```diff
@@ -359,4 +375,28 @@ public without sharing class HOT_WorkOrderHandler extends MyTriggers {
         }
         return workOrders;
     }
+
+    public static void copySubjectAndDescription(List<workOrder> workOrders) {
+        for (WorkOrder workOrder : workOrders) {
+            workOrder.HOT_UserSubject__c = workOrder.Subject;
+            workOrder.HOT_UserDescription__c = workOrder.Description;
+        }
+    }
+
+    public static void updateSubjectAndDescription(List<workOrder> workOrders, Map<Id, sObject> triggerOldMap) {
+        for (WorkOrder workOrder : workOrders) {
+            if (workOrder.Subject != triggerOldMap.get(workOrder.Id).get('Subject')) {
+                workOrder.HOT_UserSubject__c = workOrder.Subject;
+            }
+            if (workOrder.Description != triggerOldMap.get(workOrder.Id).get('Description')) {
+                workOrder.HOT_UserDescription__c = workOrder.Description;
+            }
+            if (workOrder.HOT_UserSubject__c != triggerOldMap.get(workOrder.Id).get('HOT_UserSubject__c')) {
+                workOrder.Subject = workOrder.HOT_UserSubject__c;
+            }
+            if (workOrder.HOT_UserDescription__c != triggerOldMap.get(workOrder.Id).get('HOT_UserDescription__c')) {
+                workOrder.Description = workOrder.HOT_UserDescription__c;
+            }
+        }
+    }
```

Comment:
Samme kommentarene for RequestHandler gjelder også her.
Må håndtere opprettelse fra Community, og navn på lister kan forbedres.